Mess Management System/formStudentAccount.vb

```diff
@@ -31,8 +31,7 @@ Public Class formStudentAccount
     Private Sub fill()
         Try
             con.connect()
-            Dim query As String = "SELECT student_takes.BILL_ID, student_takes.T_ID, student_takes.S_ID, item.NAME, student_takes.QTY, student_takes.DOR, student_takes.RATE, student_takes.E_ID, student_takes.TAX from student_takes INNER JOIN item ON item.ITEM_ID=student_takes.ITEM_ID  where date_format(Date(DOR),'%d-%m-%Y') like '%" + MonthCalendar1.SelectionRange.Start.Month.ToString + "-" + MonthCalendar1.SelectionRange.Start.Year.ToString + "' order by dor desc"
-            'where date_format(Date(DOR),'%d-%m-%Y') like '%" + MonthCalendar1.SelectionRange.Start.Month.ToString + "-" + MonthCalendar1.SelectionRange.Start.Year.ToString + "' ORDER BY DOR DESC"
+            Dim query As String = "create view stuacc as select student_takes.BILL_ID,student_takes.BILL_ID, student_takes.T_ID, student_takes.S_ID, item.NAME, student_takes.QTY, student_takes.DOR, student_takes.RATE, student_takes.E_ID , student_takes.TAX from student_takes , item where item.ITEM_ID=student_takes.ITEM_ID;"
             Dim comm As New MySqlCommand(query, con.conn)
```

Comment:
Hi Karan,
This is not exactly what I am looking for.

1. First update the SQL File that has the database structure. Add this statement(which is the view definition) below, at the end of that file.
Thus, view will be created once at the start while installing the database.
> CREATE VIEW stuacc AS
> SELECT student_takes_anc.BILL_ID AS BILL_ID,
>       student_takes_anc.T_ID AS T_ID,
>       student_takes_anc.S_ID AS S_ID,
>       item_anc.NAME AS NAME,
>       student_takes_anc.QTY AS QTY,
>        student_takes_anc.DOR AS DOR,
>       student_takes_anc.RATE AS RATE,
>       student_takes_anc.E_ID AS E_ID,
>       student_takes_anc.TAX AS TAX
>FROM student_takes_anc,
>     item_anc
>WHERE item_anc.ITEM_ID=student_takes_anc.ITEM_ID;

2. Here(or in fact wherever the data is queried, you would have to just query the view as)
 `SELECT * from stuacc;`

Since, you would have already created the view while setting up the system (through the SQL file), 
you would not need to create the views again, rather just query them to get the data.

Please let me know if you have any other doubts.